app/models/ext_management_system.rb

```diff
@@ -66,6 +66,7 @@ def self.api_allowed_attributes
   has_many :storage_profiles,    :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
   has_many :physical_racks,      :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
   has_many :physical_switches,   :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
+  has_many :physical_chassis,    :foreign_key => "ems_id", :class_name => "PhysicalChassis", :dependent => :destroy, :inverse_of => :ext_management_system
```

Comment:
Did you need the class_name just for the inflection?  If so you can add it to https://github.com/ManageIQ/manageiq/tree/master/lib/vmdb/inflections.rb

Reply:
The error on the screenshot happens when I remove the "**:class_name => "PhysicalChassis**"

Reply:
That is because you need the capital inflection for the class names, see Queue/queue

Reply:
Right. I was using the inflections only to the table names. Now It is for Class Names too.